tests/org.eclipse.reddeer.eclipse.test/src/org/eclipse/reddeer/eclipse/test/jdt/ui/wizards/InterfaceWizardTest.java

```diff
@@ -0,0 +1,178 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat, Inc - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.reddeer.eclipse.test.jdt.ui.wizards;
+
+import static org.junit.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.reddeer.common.exception.RedDeerException;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.JavaProjectWizard;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.NewInterfaceCreationWizard;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.NewInterfaceWizardPage;
+import org.eclipse.reddeer.eclipse.jdt.ui.wizards.NewJavaProjectWizardPageOne;
+import org.eclipse.reddeer.eclipse.ui.perspectives.JavaPerspective;
+import org.eclipse.reddeer.junit.runner.RedDeerSuite;
+import org.eclipse.reddeer.requirements.openperspective.OpenPerspectiveRequirement.OpenPerspective;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Test class for Java Interface Wizard
+ * 
+ * @author zcervink@redhat.com
+ * 
+ */
+@RunWith(RedDeerSuite.class)
+@OpenPerspective(JavaPerspective.class)
+public class InterfaceWizardTest {
+
+	protected static final String PROJECT_NAME = "testproject";
+	protected static final String INTERFACE_NAME = "InterfaceName";
+	protected static final String SOURCE_FOLDER_NAME = PROJECT_NAME + "/src";
+	protected static final String PACKAGE = "org.example.test";
+	protected static final String ENCLOSING_TYPE_CUSTOM_VALUE = "org.example.test." + INTERFACE_NAME;
+
+	@BeforeClass
+	public static void setup() {
+		createTestProject();
+	}
+
+	@Test
+	public void testInterfaceCreatedSuccessfully() {
```

Comment:
Judging by test name I would expect some actual testing of java class/interface file opened in editor. This seems more like wizard UI testing. Please, rename the test method and probably add some test for successfully created interface java file in editor. thanks.

Reply:
Renamed and split into more specific tests.

Reply:
As @odockal pointed out, its more like UI wizard testing, it would be also nice to create interface with some options and check created file, change options, change created file...I would say 1-2 files should be enough.

Reply:
Fixed. Now test is creating 2 interfaces (with different settings) and there is a check after creation of them.